dens-transactions/src/DensTransactions/scripts/mkElemIDMintingPolicy.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Is it please possible to include instructions somewhere for how to update these scripts? 


Ideally, if you're feeling energetic, I'd love it if these files were passed in via environment variables, and we'd `JSON.parse` those i.e.,
```typescript
import { readFileSync } from 'node:fs';

export const elemIdMintingPolicy = (() => {
  const ELEM_ID_MINTING_POLICY='ELEM_ID_MINTING_POLICY_PATH'
  const path = process.env[ELEM_ID_MINTING_POLICY]
  if (path === undefined)
    throw new Error(`Expected environment variable ELEM_ID_MINTING_POLICY_PATH to be a file path to a JSON object containing the script`)

  const jsonFile = JSON.parse(readFileSync(path))
  ...
})
```

Then, finally, we'd set up the nix environment s.t. it sets the environment variables correctly.

Reply:
I'll take a crack at combining nix with the exporter next week. Just pushed a readme w/ instructions to the contract tests repo, so marking this resolved.